sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/DateTimeUtils.scala

```diff
@@ -237,114 +237,114 @@ object DateTimeUtils {
    *  - Region-based zone IDs in the form `area/city`, such as `Europe/Paris`
    */
   def stringToTimestamp(s: UTF8String, timeZoneId: ZoneId): Option[Long] = {
```

Comment:
Can we make the change minimised? you can do, for example, 
```suggestion
  def stringToTimestamp(s: UTF8String, timeZoneId: ZoneId): Option[Long] = try {
```

Reply:
ok done!